### DISCUSSION TITLE
Is there as size limit or a format to the tag?

### DISCUSSION THREAD

**Discussion by tademet** [ncipollo/release-action#383] [Q&A] [answered]
I am getting this error
`Error 422: Validation Failed: {"resource":"Release","code":"custom","field":"tag_name","message":"tag_name is not a valid tag"}, {"resource":"Release","code":"custom","message":"Published releases must have a valid tag"}, {"resource":"Release","code":"invalid","field":"target_commitish"}
`
and the Tag evaluates out to KT.SFMC.CustomActivitiesAPI-13.  Is there a length limit or some format that is not working with this?

**Comment by ncipollo** [comment]
This is an error originating from GitHub (and/or git itself). I'd check the GitHub docs here to see what formatting rule you are running afoul of.

**Reply by tademet** [reply]
Can yo narrow down where I would look in the documents?

**Reply by ncipollo** [reply]
https://stackoverflow.com/questions/26382234/what-names-are-valid-git-tags

**Reply by ncipollo** [reply, marked as answer]
Also it looks like whatever you are providing for a commit is not valid. That could be the ultimate source of the issue since it would prevent a tag from being created.
